README.md

```diff
@@ -70,62 +75,39 @@ $ diesel migration run
 ```
 
-### Run NEAR Indexer for Explorer
-
-Command to run NEAR Indexer for Explorer have to contain sync mode.
-
-You can choose NEAR Indexer for Explorer sync mode by setting what to stream:
- - `sync-from-latest` - start indexing blocks from the latest finalized block
- - `sync-from-interruption --delta <number_of_blocks>` - start indexing blocks from the block NEAR Indexer was interrupted last time but earlier for `<number_of_blocks>` if provided
- - `sync-from-block --height <block_height>` - start indexing blocks from the specific block height
-
-Optionally you can tell Indexer to store data from genesis (Accounts and Access Keys) by adding key `--store-genesis` to the `run` command.
-
-NEAR Indexer for Explorer works in strict mode by default, but you can disable it for specific amount of blocks. The strict mode means that every piece of data
-will be retried to store to database in case of error. Errors may occur when the parent piece of data is still processed but the child piece is already
-trying to be stored. So Indexer keeps retrying to store the data until success. However if you're running Indexer not from the genesis it is possible that you
-really miss some of parent data and it'll be impossible to store child one, so you can disable strict mode for 1000 blocks to ensure you've passed the strong
-relation data area and you're running Indexer where it is impossible to loose any piece of data.
+NEAR Indexer for Explorer works in strict mode by default, but you can disable it. The strict mode means that every piece of data
+will be retried to store to database in case of error. Errors may occur when the parent piece of data is still processed but the child piece is already trying to be stored. So Indexer keeps retrying to store the data until success. However, if you're running Indexer not from the genesis it is possible that you really miss some of parent data and it'll be impossible to store a child one, so you can disable strict mode to ensure you've passed the strong relation data area and you're running Indexer where it is impossible to loose any piece of data.
 
 To disable strict mode you need to provide:
 
 ```
 --non-strict-mode
 ```
 
-Sometimes you may want to index block while sync process is happening, by default an indexer node is waiting for full sync to complete but you can enable indexing while the node is syncing by passing `--stream-while-syncing`
-
 By default NEAR Indexer for Explorer processes only a single block at a time. You can adjust this with the `--concurrency` argument (when the blocks are mostly empty, it is fine to go with as many as 100 blocks of concurrency).
 
 So final command to run NEAR Indexer for Explorer can look like:
 
 ```bash
-$ cargo run --release -- --home-dir ~/.near/testnet run --store-genesis --stream-while-syncing --non-strict-mode --concurrency 1 sync-from-latest
+$ ./target/release/indexer-explorer-lake \
+  --non-strict-mode \
+  --concurrency 1 \
+  from-latest
```

Comment:
This command misses the chain id. Could you add it in the proper place, please?

Reply:
Done